src/backend/marsha/bbb/tests/test_api.py

```diff
@@ -213,10 +211,10 @@ def test_api_classroom_create_student(self):
         """A student should not be able to create a classroom."""
         classroom = ClassroomFactory()
 
-        jwt_token = AccessToken()
-        jwt_token.payload["resource_id"] = str(classroom.id)
-        jwt_token.payload["roles"] = ["student"]
-        jwt_token.payload["permissions"] = {"can_update": True}
+        jwt_token = LTIStudentResourceAccessTokenFactory(
+            resource=classroom,
+            permissions__can_update=True,
```

Comment:
doing the opposite seems more intuitive now but harder to refacto. The idea would be to set `can_update` to `True` by default and change it to `False` when it's needed.